daemon/daemon_unix.go

```diff
@@ -824,11 +823,9 @@ func parseRemappedRoot(usergrp string) (string, string, error) {
 			groupname = lgrp.Name
 		} else {
 			// not a number; attempt a lookup
-			group, err := user.LookupGroup(idparts[1])
-			if err != nil {
+			if _, err := user.LookupGroup(idparts[1]); err != nil {
 				return "", "", fmt.Errorf("Error during gid lookup for %q: %v", idparts[1], err)
 			}
-			groupID = group.Gid
```

Comment:
ohmans. :) Nice catches @jfrazelle.  So the refactoring from the original userns impl. to use `subuid` & `subgid` changed this function from converting from name, group -> uid, gid to the reverse (as name, groupname were required to look up the subordinate ranges), but some cruft got left in as you found :-/